topics/ecmascript/index.md

```diff
@@ -0,0 +1,12 @@
+---
+topic: ecmascript
+display_name: ECMAScript
```

Comment:
Could we please keep these key-values alphabetized? This will be in line with most other topics in the repo.

Reply:
ECMAScript